import os
import sys


def err_exit(message):
    print(message)
    sys.exit(1)